### PR TITLE
v2.5: Docs!

### PR DESCRIPTION
**Pull request by NickCraver**
This is a pass at updating the docs for the 2.5 prerelease - thoughts?

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Release Notes
 
-## Unreleased
+## 2.5.x-prerelease
 
+- Adds a backlog/retry mechanism for commands issued while a connection isn't available (#1912 via NickCraver)
+  - Commands will be queued if a multiplexer isn't yet connected.
```
Comment (philon-msft) [inline docs/ReleaseNotes.md:6]: "...isn't yet connected to a Redis server"?

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+  - Commands will be queued if a connection is lost, for retry when it reconnects.
```
Comment (philon-msft) [inline docs/ReleaseNotes.md:7]: semantic nit: it's not exactly "retry"... more like "Commands will be queued if a connection is lost, and then sent to the server when the connection is restored"

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -1,7 +1,12 @@
 # Release Notes
 
-## Unreleased
+## 2.5.x-prerelease
```
Comment (philon-msft) [inline docs/ReleaseNotes.md:3]: "2.5.27-prerelease"?

**Review by philon-msft — APPROVED** [review]

**Comment by NickCraver** [comment]
@philon-msft good suggestions - thanks! Updated for one more round of eyes :)

**Pull request merged**